### ISSUE TITLE
Unit Test result when grading w/ Multiple Selections shows up in the "wrong" place

### ISSUE THREAD

**Issue by psmaker** [RunestoneInteractive/RunestoneServer#1368]
Course: LHSIntroCS_2019
Page: https://runestone.academy/runestone/admin/grading
Bug: When grading manually with multiple selections, when I run a test for someone other than the first person, the GREEN/RED table of results shows up in the 1st person's area, not the actual student in which I pressed the RUN button.

![image](https://user-images.githubusercontent.com/28868409/67032335-3da72380-f0c8-11e9-8089-4b5aac0d8d11.png)

**Comment by bnmnetp** [comment]
The challenge of displaying multiple students and some of the assumptions built into the naming conventions of the activecode windows ended up in a catastrophic collision on this one.

I heard about this one on slack yesterday, so a solution is slowly starting to emerge in my mind.  But it may take a couple of days to get it coded and debugged as this one goes pretty deep.

**Comment by bnmnetp** [comment]
This was fixed in yesterday's release.

**Comment by psmaker** [comment]
It's looking good! I do notice an minor odd behavior.

I tried selecting one assignment problem and 4 students. I pressed 'Run' on the 4th student. The view scrolled me to the top of the screen. I scrolled back down to the 4th student and no results were shown, nor evidence that the unittest was even run (as my unit test should print out "Auto-testing..."). Then I pressed 'Run' again and it works just as expected - the unittest runs and the results are shown. The view stays on this students (not scrolling back to the top).

I tried this with 4 students and choosing the 2nd or 3rd student. Same type of results as above.

But if I try the 1st student, everything is OK.

Something about clicking 'Run' on the **non-first** student in the list the **first time**. Second time works OK.
